src/renderer/base/FontCache.h

```diff
@@ -62,13 +62,16 @@ namespace Microsoft::Console::Render::FontCache
 
                 wil::com_ptr<IDWriteFontSetBuilder1> fontSetBuilder;
                 THROW_IF_FAILED(factory5->CreateFontSetBuilder(fontSetBuilder.addressof()));
-                THROW_IF_FAILED(fontSetBuilder->AddFontSet(systemFontSet.get()));
 
                 for (const auto& file : nearbyFontFiles)
                 {
                     LOG_IF_FAILED(fontSetBuilder->AddFontFile(file.get()));
                 }
 
+                // IDWriteFontSetBuilder ignores any families that have already been added.
```

Comment:
Critically, this is true on Windows 10, Right?

Reply:
Yep, tested!